ansible/xq-api.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Since this really only makes sense when it has access to a mirror, lets add it to the buildmaster instead.

Reply:
Removed and added under buildmaster.